resources/views/achievement/gallery.blade.php

```diff
@@ -8,7 +8,7 @@
 
     <div id="achievement-accordion">
 
-        <?php $stars = 1; ?>
+        <?php $stars = 0; ?>
```

Comment:
This variable seems totally unnecessary, I think it can be removed (including the ++ at the end)